pkgs/development/python-modules/python-matter-server/default.nix

```diff
@@ -26,6 +28,28 @@
 , pytestCheckHook
 }:
 
+let
+  paaCerts = stdenvNoCC.mkDerivation {
+    name = "matter-server-paa-certificates";
```

Comment:
```suggestion
    pname = "matter-server-paa-certificates";
    version = "1.2.0.1";
```

Reply:
Thanks for pointing this out, I had to search what the difference between pname and name is, but I think I understand now.